docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -15,7 +15,85 @@ keywords: [Databricks, destination, data warehouse]
 pip install dlt[databricks]
 ```
 
-## Setup Guide
+## Set up your Databricks workspace
+
+To use the Databricks destination, you need:
+
+* A Databricks workspace with a Unity Catalog metastore connected
+* A Gen 2 Azure storage account and container
+
+If you already have your Databricks workspace set up, you can skip to the [Loader setup Guide](#loader-setup-guide).
+
+### 1. Create a Databricks workspace in Azure
+
+1. Create a Databricks workspace in Azure
+
+    In your Azure Portal search for Databricks and create a new workspace. In the "Pricing Tier" section, select "Premium" to be able to use the Unity Catalog.
+
+2. Create a storage account
+
+    Search for "Storage accounts" in the Azure Portal and create a new storage account. Make sure to select "StorageV2 (general purpose v2)" as the account kind.
```

Comment:
In this step the user needs to make sure he is creating an ADLS Gen 2 storage account (a specific type of Standard general-purpose v2 storage account), which can be done by enabling the hierarchical namespace. Probably best to just refer to Azure's docs for this: https://learn.microsoft.com/en-us/azure/storage/blobs/create-data-lake-storage-account